tests/api2/test_auth_failover.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from middlewared.test.integration.utils import call, password, truenas_server
+from truenas_api_client import Client
+
+import sys
+import os
+apifolder = os.getcwd()
+sys.path.append(apifolder)
```

Comment:
Do we really need this `sys.path` management?

Reply:
Fixed